crates/miden-objects/src/note/note_tag.rs

```diff
@@ -570,7 +624,7 @@ mod tests {
     #[test]
     fn test_for_private_use_case() {
         let tag = NoteTag::for_local_use_case(0b0, 0b0).unwrap();
-        assert_eq!(tag, NoteTag(0b11000000_00000000_00000000_00000000));
+        assert_eq!(tag, NoteTag::from(0b10000000_00000000_00000000_00000000));
```

Comment:
I think _the test_ was correct before. `NoteTag::for_local_use_case` should return `NoteTag::LocalUseCase` and its prefix is _now_ `0b11`. 

Previously, we already returned a tag with `0b11` prefix from `NoteTag::for_local_use_case`, even though the prefix should have been `0b10`, afaict. Because we've swapped `LocalAccount` with `LocalUseCase`, the `0b11` prefix should now be the correct one here.